proxy/envoy/watcher.go

```diff
@@ -159,11 +159,13 @@ func (w *watcher) UpdateIngressSecret(ctx context.Context) error {
 		return multierror.Prefix(err, "failed to read request body")
 	}
 	if len(tlsData) == 0 {
+		glog.Errorf("failed to get TLS data, zero data")
 		return nil
 	}
 
 	var tls model.TLSSecret
 	if err = json.Unmarshal(tlsData, &tls); err != nil {
+		glog.Errorf("failed to unmarshal TLS secret")
```

Comment:
nit: this might be very noisy in the test logs

Reply:
Indeed, I get plenty of these fails, I will see if I can set it to a verbosity level. I wonder why we get them, do they indicate a real issue?